source/backend/api/Areas/Acquisition/Controllers/ExpropriationPaymentController.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using MapsterMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Pims.Api.Models.Concepts;
+using Pims.Api.Policies;
+using Pims.Api.Services;
+using Pims.Core.Extensions;
+using Pims.Core.Json;
+using Pims.Dal.Entities;
+using Pims.Dal.Security;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Pims.Api.Areas.Acquisition.Controllers
+{
+    /// <summary>
+    /// Form8Controller class, provides endpoints for Acquisition File's form 8's.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Area("acquisitionfiles")]
+    [Route("v{version:apiVersion}/[area]")]
+    [Route("[area]")]
+    public class ExpropriationPaymentController : ControllerBase
+    {
+        private readonly IAcquisitionFileService _acquisitionFileService;
+        private readonly IMapper _mapper;
+        private readonly ILogger _logger;
+
+        public ExpropriationPaymentController(IAcquisitionFileService acquisitionService, IMapper mapper, ILogger<ExpropriationPaymentController> logger)
+        {
+            _acquisitionFileService = acquisitionService;
+            _mapper = mapper;
+            _logger = logger;
+        }
+
+        /// <summary>
+        /// Get all Expropriation Payments from the Acquisition File.
+        /// </summary>
+        /// <param name="id">Acquisition File Id.</param>
+        /// <returns>List of all Expropriation Payments created for the acquisition file.</returns>
+        [HttpGet("{id:long}/expropriation-payment")]
```

Comment:
can we pluralize this and all swagger refs? expropriation-payment -> expropriation-payments

Reply:
Updated